requirements.txt

```diff
@@ -5,3 +5,4 @@ matplotlib==2.0.0
 intervaltree==2.1.0
 sphinx==1.5.2
 jinja2==2.9
+pybedtools==0.7.10
```

Comment:
make sure to add this to setup.py as well.

Reply:
Done. Also added myself as an author, as I feel I have contributed enough to warrant so. Let me know if this is an issue.